dockstore-webservice/src/main/java/io/dockstore/webservice/resources/DockerRepoResource.java

```diff
@@ -980,7 +980,10 @@ public void checkCanRead(User user, Entry tool) {
         FileType fileType =
             (descriptorType.toUpperCase().equals(DescriptorType.CWL.toString())) ? DescriptorLanguage.FileType.CWL_TEST_JSON : DescriptorLanguage.FileType.WDL_TEST_JSON;
         for (String path : testParameterPaths) {
-            sourceFiles.removeIf((SourceFile v) -> v.getPath().equals(path) && v.getType() == fileType);
+            boolean fileDeleted = sourceFiles.removeIf((SourceFile v) -> v.getPath().equals(path) && v.getType() == fileType);
```

Comment:
Based on a discussion on another PR, I wonder if this should be case-insensitive? Probably better if not, even if we're slightly inconsistent.

Reply:
:thinking: